src/sprites/rendered-target.js

```diff
@@ -194,6 +209,18 @@ class RenderedTarget extends Target {
         return "don't rotate";
     }
 
+    /**
+     * Available states for video input.
+     * @type {object}
+     */
+    static get VIDEO_STATE () {
+        return {
+            'OFF': 'off',
+            'ON': 'on',
+            'ON-FLIPPED': 'on-flipped'
```

Comment:
`ON-FLIPPED` cannot be used like "normal" enums. `VIDEO_STATE.ON-FLIPPED` will be a ReferenceError.

Reply:
@mzgoddard Is this because of the hyphen? or am I missing something?

Reply:
Fixed in #1031